aries_vcx/src/handlers/issuance/issuer/issuer.rs

```diff
@@ -32,6 +34,42 @@ pub enum IssuerState {
     Failed,
 }
 
+
+fn _append_credential_preview(cred_offer_msg: CredentialOffer, credential_json: &str) -> VcxResult<CredentialOffer> {
```

Comment:
This was moved from its original location handlers/issuance/issuer/state_machine.rs